app/api/apiConfig.js

```diff
@@ -476,6 +486,14 @@ export const settingsAPIs = {
             location: "Paris",
             operator: "Witness: zapata42-witness",
             contact: "telegram:Zapata_42"
+        },
+        {
+            url: "wss://citadel.li/node",
+            location: "Island"
+        },
+        {
+            url: "ws://citadel2miawoaqw.onion/node",
```

Comment:
How do you use a Tor node with the UI?

Reply:
via a Tor Browser)
it'll be our local feature, so i removed it for now since it needs to be fixed
(there're some problems with indexeddb connection in Tor)